docs/fastdds/dds_layer/topic/topic/topic.rst

```diff
@@ -2,3 +2,200 @@
++------------------------------+----------------------------+----------+
+| LatencyBudgetQosPolicy       | :func:`latency_budget`     | Yes      |
++------------------------------+----------------------------+----------+
+| LivelinessQosPolicy          | :func:`liveliness`         | Yes      |
++------------------------------+----------------------------+----------+
+| ReliabilityQosPolicy         | :func:`reliability`        | Yes      |
++------------------------------+----------------------------+----------+
+| DestinationOrderQosPolicy    | :func:`destination_order`  | Yes      |
++------------------------------+----------------------------+----------+
+| HistoryQosPolicy             | :func:`history`            | Yes      |
++------------------------------+----------------------------+----------+
+| ResourceLimitsQosPolicy      | :func:`resource_limits`    | Yes      |
++------------------------------+----------------------------+----------+
+| TransportPriorityQosPolicy   | :func:`transport_priority` | Yes      |
++------------------------------+----------------------------+----------+
+| LifespanQosPolicy            | :func:`lifespan`           | Yes      |
++------------------------------+----------------------------+----------+
+| OwnershipQosPolicy           | :func:`ownership`          | Yes      |
++------------------------------+----------------------------+----------+
+| DataRepresentationQosPolicy  | :func:`representation`     | Yes      |
++------------------------------+----------------------------+----------+
+
+Refer to the detailed description of each :class:`QosPolicy` class for more information about their usage and
+default values.
+
+The QoS value of a previously created :ref:`dds_layer_topic_topic` can be modified using the
+:func:`set_qos()` member function.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_TOPICQOS
+   :end-before: //!
+
+
+.. _dds_layer_defaultTopicQos:
+
+Default TopicQos
+^^^^^^^^^^^^^^^^
+
+The default :ref:`dds_layer_topic_topicQos` refers to the value returned by the
+:func:`get_default_topic_qos()` member function on the :ref:`dds_layer_domainParticipant` instance.
+The special symbol :class:`TOPIC_QOS_DEFAULT` can be used as QoS argument on :func:`create_topic()`
+or :func:`set_qos()` member functions to indicate that the current default :ref:`dds_layer_topic_topicQos`
+should be used.
+
+When the system starts, the default :ref:`dds_layer_topic_topicQos` is equivalent to the default constructed
+value :func:`TopicQos()`.
+The default :ref:`dds_layer_topic_topicQos` can be modified at any time using the
+:func:`set_default_topict_qos()` member function on the :ref:`dds_layer_domainParticipant` instance.
+Modifying the default :ref:`dds_layer_topic_topicQos` will not affect already existing :ref:`dds_layer_topic_topic`
+instances unless their QoS is modified with :func:`set_qos(TOPIC_QOS_DEFAULT)`.
```

Comment:
The trailing parenthesis is still there because the func role hasn't changed. 
If you want to set the argument to the function you can use the code role.

:code:`set_qos(TOPIC_QOS_DEFAULT`)